crm_schemas.go

```diff
@@ -0,0 +1,106 @@
+package hubspot
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	crmSchemasPath = "schemas"
+)
+
+type CrmSchemaAssociation struct {
+	Cardinality        string     `json:"cardinality"`
+	CreatedAt          *time.Time `json:"createdAt"`
```

Comment:
Like CrmProperty, CreatedAt and UpdatedAt are easier to see when they are side by side.